packages/web/index.js

```diff
@@ -30,7 +30,13 @@ module.exports = (neutrino, opts = {}) => {
     polyfills: {
       async: true
     },
-    babel: {}
+    babel: {},
+    minify: {
+      minify: {
```

Comment:
There's a double nesting here.

Reply:
This is intentional, as options.minify is what gets passed to the preset, which has options { minify, overrides}

Reply:
Maybe we shouldn't allow overrides here to keep things simple? I could definitely see users getting confused about the double-word nesting here.